lib/components/Modal/bottom/modal-bottom.stories.tsx

```diff
@@ -0,0 +1,474 @@
+        },
+        isOpened: {
+            table: { type: { summary: "boolean | undefined" } },
+            options: ["true", "false"],
+            description: "Controls the visibility of the modal",
+            control: { type: "boolean" },
+        },
+        className: {
+            table: { type: { summary: "string | undefined" } },
+            description: "ClassName for external tag of the component.",
+            control: { type: "text" },
+        },
+        showHandleBar: {
+            table: { type: { summary: "boolean | undefined" } },
+            options: ["true", "false"],
+            description:
+                "Controls the visibility of the handlebar. If handlebar is visible, the modal can be expanded/collapsed by swiping up/down handlebar. Default value is true.",
+            control: { type: "boolean" },
+        },
+        showSecondaryButton: {
+            table: { type: { summary: "boolean | undefined" } },
+            options: ["true", "false"],
+            description: "Controls the visibility of the secondary button.",
+            control: { type: "boolean" },
+        },
+        shouldCloseOnPrimaryButtonClick: {
+            table: { type: { summary: "boolean | undefined" } },
+            options: ["true", "false"],
+            description:
+                "Flag for controlling modal behavior. If it's true, then the modal will be closed after user clicks on the primary button.",
+            control: { type: "boolean" },
+        },
+        toggleModal: {
+            table: { type: { summary: "(isOpened: boolean) => void" } },
+            description:
+                "Function for changing state of the visibility of the modal.",
+            control: { type: null },
+        },
+        portalId: {
+            table: { type: { summary: "string | undefined" } },
+            description:
+                "ID of the modal container. If it wasn't passed, the modal will try to find any container with 'modal-root' ID. If there is no container with 'modal-root' ID the modal will be insert inside document.body.",
+            control: { type: "string" },
+        },
+        primaryButtonLabel: {
+            table: { type: { summary: "ReactNode" } },
+            description: "Label for the primary button.",
+            control: { type: "string" },
+        },
+        primaryButtonCallback: {
+            table: { type: { summary: "(() => void) | undefined" } },
+            description:
+                "Function which will be called on clicking on primary button.",
+            control: { type: null },
+        },
+        secondaryButtonLabel: {
+            table: { type: { summary: "ReactNode | undefined" } },
+            description: "Label for the secondary button.",
+            control: { type: "string" },
+        },
+    },
+} satisfies Meta<typeof ModalBottom>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const DefaultModalBottom: Story = {
```

Comment:
im seeing duplicates here. Think its better to define template work off of it so you dont have to replicate same logic in each variant.

https://www.youtube.com/watch?v=FUKpWgRyPlU

starts at minute 11

Reply:
Thank you, let me have a look

Reply:
Worked perfectly, super, thank you)